fonduer/__init__.py

```diff
@@ -1,52 +1,66 @@
 import logging
 
 from fonduer._version import __version__
-from fonduer.annotations import load_gold_labels
-from fonduer.async_annotations import (BatchFeatureAnnotator,
-                                       BatchLabelAnnotator)
 from fonduer.candidates import CandidateExtractor, OmniFigures, OmniNgrams
+from fonduer.candidates.matchers import (
```

Comment:
These are just following convention and are alphabetical. I wouldn't change them.

Reply:
Also this whole file will look very very different after the next PR when we pull imports into submodules instead of having everything right at the root.